### PR TITLE
Add legacy frontend flag to the OpenVINO conversion

### PR DESCRIPTION
**Pull request by HonzaCuhel**

### CODE REVIEW

File: yolo/exporter.py [hunk 1]
```diff
@@ -61,6 +61,7 @@ def export_openvino(self, version):
         f"--model_name '{self.model_name}' " \
         '--data_type FP16 ' \
         '--reverse_input_channels ' \
+        '--use_legacy_frontend ' \
```
Comment (tersekmatija) [inline yolo/exporter.py:64]: Can we make this togglable under advanced options for both exporters?

File: requirements.txt [hunk 2]
```diff
@@ -32,3 +32,6 @@ sentry-sdk # logging
 thop  # FLOPs computation
 
 sparseml==1.2.0
+
+websockets==10.4
+openvino-dev[onnx]
```
Comment (tersekmatija) [inline requirements.txt:37]: Might be good to specify 2022.1 or 2022.3 version here just in case.

**Review by tersekmatija — CHANGES_REQUESTED** [review]

File: yolo/exporter.py (hunk 1 above)
Comment (HonzaCuhel) [reply yolo/exporter.py:64]: So you mean like option (checkbox) for using the using the legacy flag?

Comment (tersekmatija) [reply yolo/exporter.py:64]: Yeah exactly.

File: requirements.txt (hunk 2 above)
Comment (HonzaCuhel) [reply requirements.txt:37]: But if I remember correctly, the blobconverter doesn't support OpenVINO 2022.3, the latest version OpenVINO that is supports is 2022.1

Comment (tersekmatija) [reply requirements.txt:37]: Yeah, it's not yet updated. Let's set it to 2022.1 :)

Comment (HonzaCuhel) [reply requirements.txt]: Updated

**Review by tersekmatija — APPROVED** [review]

**Comment by tersekmatija** [comment]
LGTM, thanks! Feel free to merge and deploy.

**Comment by HonzaCuhel** [comment]
Thanks!

**Pull request merged**